src/test/resources/assets/ambient_occlusion_elements_test/models/block/ambient_occlusion.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parent": "block/block",
+  "render_type": "cutout",
+  "textures": {
+    "all": "minecraft:block/white_concrete",
+    "overlay": "minecraft:block/glass"
+  },
+  "elements": [
+    {
+      "from": [ 0, 0, 0 ],
+      "to": [ 16, 16, 16 ],
+      "shade": false,
```

Comment:
This does not work. Check your own deserialized again.

Reply:
I just tested it and it worked for me. But i can see that it's hard to see the difference, so i'll expand test mod to have a 4 blocks with different shade and ao settings and pure white texture so the result will be more clear to see and compare